bin/oneoff/behavior_migration.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+require_relative '../../dashboard/config/environment'
+
+GamelabJr.all.each do |level|
+  next unless level.custom_blocks
+  blocks = JSON.parse level.custom_blocks
+  blocks.each do |block|
+    if block['returnType'] && block['returnType'].downcase == 'function'
+      block['returnType'] = 'Behavior'
+    end
+    next unless block['args']
+    block['args'].each do |arg|
+      if arg['name'] == 'BEHAVIOR'
+        arg['type'] = 'Behavior'
+      end
+    end
+  end
+  File.write('/tmp/blocks-formatter.js',  "console.log(JSON.stringify(#{JSON.pretty_generate blocks}, null, 2))")
+  new_blocks = `node /tmp/blocks-formatter.js`
```

Comment:
why this and not `JSON.pretty_generate`?

Reply:
There were some minor formatting differences in the generated JSON, which made it harder to manually verify the diff. e.g. for some reason `args: []` became:
```
args: [

]
```